tensorflow/compiler/xla/service/gpu/BUILD

```diff
@@ -399,9 +401,20 @@ filegroup(
     ),
 )
 
+# virtual targets since nested select statements not possible
+tf_cuda_library(
```

Comment:
Would an alias with an empty stub work instead?

Reply:
The code here was mostly copied from the RCCL support in `tensorflow/core/kernels/BUILD`, see https://github.com/tensorflow/tensorflow/blob/2e98e890918e0b51727b3d76edd72922bfe7184b/tensorflow/core/kernels/BUILD#L202-L236 .

I'm not familiar with the alias and empty stub approach.  Do you have sample code, or a doc link?

Reply:
Something like this?
```
cc_library(name = "empty")
alias (
  name = "virtual_nccl",
  actual = if_cuda("@local_config_nccl//:nccl", ":empty"),
)
alias (
  name = "virtual_rccl",
  actual = if_rocm("@local_config_rocm//rocm:rccl", ":empty"),
)
```

Reply:
Thank you for the example.  Pushed the change.